bedrock/firefox/templates/firefox/includes/schemaorg-app.html

```diff
@@ -1,34 +0,0 @@
-<div itemscope itemtype="http://schema.org/SoftwareApplication">
-  <meta itemprop="name" content="{{_('Firefox')}}">
-  <meta itemprop="description" content="{% block product_desc %}{{_('Mozilla Firefox, free web browser, is created by a global non-profit dedicated to putting individuals in control &amp; shaping the future of the web for the public good.')}}{% endblock %}">
-  <meta itemprop="url" content="{{ settings.CANONICAL_URL }}/firefox/new/">
-  <meta itemprop="image" content="{% block product_logo %}{{ static('img/firefox/firefox-256.jpg') }}{% endblock %}">
-  <div itemprop="author" itemscope itemtype="http://schema.org/Organization">
-    <meta itemprop="name" content="{{_('Mozilla')}}">
-  </div>
-  <div itemprop="offers" itemscope itemtype="http://schema.org/Offer">
-    <meta itemprop="price" content="0">
-  </div>
-  <meta itemprop="softwareVersion" content="{{ latest_firefox_version }}">
-  <meta itemprop="releaseNotes" content="{{ settings.CANONICAL_URL }}/firefox/{{ latest_firefox_version }}/releasenotes/">
-  <meta itemprop="applicationCategory" content="BrowserApplication">
-  <meta itemprop="operatingSystem" content="Windows">
-  <meta itemprop="operatingSystem" content="Mac">
-  <meta itemprop="operatingSystem" content="Linux">
-  <meta itemprop="operatingSystem" content="Android">
-  <meta itemprop="operatingSystem" content="iOS">
-</div>
-
-<div itemscope itemtype="http://schema.org/Product">
-  <meta itemprop="name" content="{{_('Firefox')}}">
-  <meta itemprop="description" content="{{ self.product_desc() }}">
-  <meta itemprop="url" content="{{ settings.CANONICAL_URL }}/firefox/new/">
-  <meta itemprop="image" content="{{ self.product_logo() }}">
-  <meta itemprop="logo" content="{{ self.product_logo() }}">
-  <div itemprop="manufacturer" itemscope itemtype="http://schema.org/Organization">
-    <meta itemprop="name" content="{{_('Mozilla')}}">
-  </div>
-  <div itemprop="offers" itemscope itemtype="http://schema.org/Offer">
-    <meta itemprop="price" content="0">
-  </div>
-</div>
```

Comment:
We're good dropping this file? Is it no longer useful?

Reply:
I pondered this also. My personal view on schema.org is that it isn't really that useful (happy to be proven wrong with some measurable results). That aside though, the current quantum template is missing the strings used here, so we can't just bolt it back on. I notice there is a new task in Taiga to add metadata for the Firefox logo etc, so perhaps we can revisit this then. Thoughts?

Reply:
Hmm, perhaps I could ressurect this file using some of the existing strings in the new template. Let me have a look 👍